conf/machine/include/imx-base.inc

```diff
@@ -353,6 +357,9 @@ KERNEL_IMAGETYPE_aarch64 = "Image"
 
 MACHINE_FEATURES = "usbgadget usbhost vfat alsa touchscreen"
 
+MACHINE_FEATURES_append_imx8mqevk = " jailhouse"
+MACHINE_FEATURES_append_imx8mmevk = " jailhouse"
```

Comment:
@panfir How about the support for `imx8mnevk`?

Commit aa8331679397a0ee699070e1de69252ccfbbf41b in the source repo suggests that the support for Nano is provided.

Can you please include it also here, since we already have Nano machine?

Thanks!

-- andrey

Reply:
That is right, there was added support for the entire i.MX8M family, but I'm not sure how well it was tested for 8MN, I will come back here.
Do you have a certain use case for imx8mn? If so please let me know.

Reply:
> That is right, there was added support for the entire i.MX8M family, but I'm not sure how well it was tested for 8MN, I will come back here.

That would be really nice, thanks a lot!

> Do you have a certain use case for imx8mn? If so please let me know.

For the moment not, but when I saw that you've added support for both MQ and MM, leaving MN on the side I was wondering if there is something missing in the Nano part of the build... Since the layer contains Nano machine description and the source code for Nano is in jailhouse repo - it would be better to have the support here as well, so that people which are using Nano as a platform and interested in jailhouse hypervisor could just build it with those recipes. If you would be able to provide this support here with this PR - that would be really great!

-- andrey